pre-fixed TF_VAR to all environment variables.

asubmani opened the pull request:
If a shell variable needs to be made accessible to Terraform, it must begin with TF_VAR (in caps). Changed environment variables in the sample shellscript and added TIP on this requirement. Please refer Terraform documentation at https://www.terraform.io/docs/configuration/variables.html

azurecla commented:
Hi __@lodipalm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@JennieHubbard 
 Jennie, please review this PR and let me know if I should merge. Thanks.

rmca14 commented:
@barbkess 
FYI

JennieHubbard commented:
Approved!

asubmani committed:
pre-fixed TF_VAR to all environment variables.

If a shell variable needs to be made accessible to Terraform, it must begin with TF_VAR (in caps). Changed environment variables in the sample shellscript and added TIP on this requirement. Please refer Terraform documentation at https://www.terraform.io/docs/configuration/variables.html
